src/main/java/target/HttpTarget.java

```diff
@@ -29,7 +29,7 @@ public HttpTarget(final TargetRetryPolicy retryPolicy) {
         final var request = HttpRequest
             .newBuilder()
             .version(HttpClient.Version.HTTP_1_1)
-            .uri(URI.create(Config.TARGET_BASE_URL + Config.TOPICS_ROUTES.get(record.topic())))
+            .uri(URI.create(Config.TARGET_BASE_URL + Config.TOPICS_ROUTES.getRoute(record.topic())))
```

Comment:
This shouldn't come from config but from class TopicRoutes
The class should be injected in HttpTarget constructor, and initialized in Main with a Map topicRoutes